pandas/core/ops.py

```diff
@@ -135,7 +135,20 @@ def rfloordiv(left, right):
 
 
 def rmod(left, right):
-    return right % left
+    # cannot use % as an operand becaue this
```

Comment:
cc @jbrockmendel

Reply:
Good catch; that's an annoying corner case.  Would it be simpler to just raise if `right` is stringy?

Is there a test for this?

Typo becaue --> because

Reply:
no existing tests AFAICT, just some ones I added to test non-senscial ops.